sql/mob_groups.sql

```diff
@@ -7927,6 +7927,7 @@ INSERT INTO `mob_groups` VALUES (51,3148,118,'Pixie',0,128,2001,0,0,51,54,0);
 INSERT INTO `mob_groups` VALUES (52,2213,118,'Ketos',1,0,0,0,0,1,1,0);
 INSERT INTO `mob_groups` VALUES (53,4734,118,'Botulus_Rex',0,128,0,0,9999,94,95,0);
 INSERT INTO `mob_groups` VALUES (54,1642,118,'Goblin_Bounty_Hunter',0,0,1030,0,0,17,20,0);
+INSERT INTO `mob_groups` VALUES (55,5736,118,'Backoo',0,128,3140,2500,0,37,37,0);
```

Comment:
Might need a different pool. This is the pool for Koropokkur, which has a skill list of _only_ Wild Oats.

Reply:
Swapped him to Tom Tit Tat's pool, who uses no special abilities outside of standard mandy moves.

Reply:
but then its using all the same pool data as Tom, which may not be correct. More than likely a flag value will be different (probably size bits) and those are stored in pool.